### DISCUSSION TITLE
slack channel

### DISCUSSION THREAD

**Discussion by frofro1979** [TanStack/table#2871] [General]
Is the slack channel still active? I'm not able to find it

**Comment by Atirinsaf** [comment]
Hi there,
I have used React-table 7 datagrid with few customizations , where I have hidden all headers and just using a single column as a narrow hierarchy component. I have implemented server side sorting on this datagrid(narrow hierarchy component) by clicking on a button (This button is in a component from where I am calling react-table datagrid) . On initial render, I can see all icons in a collapsed format, when i click on this expand icon, children level API call is getting made and children data got appended post api call response. When I click on sort button, sorting order changes (eg:- "ASC","DESC") , then I see rows position which I have opened in previous sort order , those positions are showing expandable icon as expanded rather then collapsed. Can somebody help me here???

Expected Behaviour:-
I want when I click sort button, new data comes, then all rows should have the expandable icon in a collapsed form.
